src/cast_core/test/StreamStatsReporter_test.cpp

```diff
@@ -56,7 +56,9 @@ TEST_CASE_METHOD(MongoTestFixture, "StreamStatsReporter successfully connects to
         try {
             auto startCommand = bsoncxx::builder::stream::document{}
                 << "streams_startStreamProcessor" << ""
+                << "tenantId" << "test"
                 << "name" << "sp"
+                << "processorId" << "test_spid"
```

Comment:
Shouldn't this be the same as `name` ?

Reply:
I'm fairly sure this test isn't running in CI because we don't have an integration test setup for it yet. Otherwise I am a little confused as to how this is passing.

Reply:
processorId can be anything, our stream processor does not require it to be the same as the name. Let me get a successful patch build to resolve your concerns.

Reply:
Alright here is a successful patch build: 
https://spruce.mongodb.com/version/66352ff5c0fc690007de5172/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

I made a change since the last revision. Now I specify processorId and also options.featureFlags fields which are required fields.

Reply:
Hi @sandeepgd, I tried to enable this test as part of #1061, but this test failed and I couldn't find a way to fix it, so I decided to [leave it disable](https://github.com/mongodb/genny/pull/1061/commits/b94c417c4140379338f20fca576e6a3d82f12f02). If possible, could you try to enable and fix this test? Thank you!

Reply:
@PyGeek03 I see! Would you be able to file a separate ticket for this with some context? I am not the expert on how our perf tests are set up. @harendra-c would be able to look into the ticket separately.